src/wayland/generated/idle-inhibit-unstable-v1_wrapper.h

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Since you started this PR, we stopped committing generated code to git and deleted `src/wayland/generated`. The genereated files can be removed (they're now auto-built in the build directory)